github/search.go

```diff
@@ -54,6 +55,36 @@ func (s *SearchService) Repositories(query string, opt *SearchOptions) (*Reposit
 	return result, resp, err
 }
 
+type SingleCommitResult struct {
```

Comment:
golint issue here, exported struct `SingleCommitResult` should have documentation.